src/main/java/org/terasology/blockNetwork/BlockNetworkUtil.java

```diff
@@ -40,6 +53,13 @@ private static Rotation getRotation(Block block) {
         return rotation;
     }
 
+    /**
+    * Give the new sides of the block based on its rotation
```

Comment:
@pollend @jellysnake What do you think about the use of the word `give` here? Can we replace it with `gives` or `returns` ?

Reply:
oh yes i mean it gives...